MonumentAddons.cs

```diff
@@ -5619,11 +5649,22 @@ private void UpdateSpawnPointReferences()
                 }
             }
 
+            private void UpdateSpawnPointPositions()
+            {
+                foreach (var adapter in Adapters)
+                {
+                    if (!adapter.IsAtIntendedPosition)
+                        adapter.SpawnPoint.transform.SetPositionAndRotation(adapter.IntendedPosition, adapter.IntendedRotation);
```

Comment:
Rather than reaching into the SpawnPoint transform, I suggest we expose a method on the `SpawnPointAdapter` class to handle the movement. That is where logic can be added to move the spawned entities as well.